.github/workflows/pylint.yml

```
@@ -21,11 +23,19 @@ jobs:
         python get-pip.py
         pip install pylint==1.1.0
         pip install astroid==1.2
-        
+
     - name: Run Pylint Checks
       run: |
```

```suggestion
      run: ${{ interpreted_pylint }}
```
You can try adding this to the second line, below `name: pylint`:
```
interpreted_pylint: |
    function interpreted_pylint() {
        local error_code=0
        pylint --rcfile=.pylintrc convert2rhel/ || error_code=$?
        if ! [ $((error_code & 3)) -eq 0 ]; then
            return $error_code;
        fi;
        return 0;
     }
     interpreted_pylint
```

---

I guess it's not that important..